front/app/containers/ProjectsShowPage/timeline/VotingResults/ProgressBar/ProgressBarWrapper.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { ReactNode } from 'react';
+// components
+import { Box, Text, stylingConsts } from '@citizenlab/cl2-component-library';
+import Tippy from '@tippyjs/react';
+
+// styling
+import { useTheme } from 'styled-components';
+import { transparentize } from 'polished';
+
+interface Props {
+  votesPercentage: number;
+  children: ReactNode;
+  tooltip: string;
+}
+
+const ProgressBarWrapper = ({ children, votesPercentage, tooltip }: Props) => {
+  const theme = useTheme();
+
+  return (
+    <Tippy
+      disabled={!tooltip}
```

Comment:
```suggestion
      disabled={false}
```

`tooltip` seems to be always defined, so `!tooltip` should always evaluate to `false`